tidb-cloud/serverless-faqs.md

```diff
@@ -20,29 +20,56 @@ TiDB Serverless offers the TiDB database with full HTAP capabilities for you and
+To estimate the RU consumption of individual SQL statements, you can use the [EXPLAIN ANALYZE](/sql-statements/sql-statement-explain-analyze.md) SQL statement. You can see the RUs and storage your cluster has used in the Usage this month section of the Cluster Overview page. With past resource usage data and real-time resource usage graphs available on the Usage this month page, you can establish a reasonable spend limit for your cluster with the Edit Spend Limit feature. See [Manage Spend Limit for TiDB Serverless clusters](/tidb-cloud/manage-serverless-spend-limit.md) for more information.
 
-### What are the limitations of a TiDB Serverless cluster in beta?
+### How the storage is being metered for TiDB Serverless？
 
-For each organization in TiDB Cloud, you can create a maximum of five TiDB Serverless clusters by default. To create more TiDB Serverless clusters, you need to add a credit card and set a [spend limit](/tidb-cloud/tidb-cloud-glossary.md#spend-limit) for the usage.
+The storage is metered based on the amount of data stored in the TiDB cluster, measured in GB-months. This calculation is based on the sum of the size of all the tables and indexes in the cluster, without including data compression or replicas for high availability, and is multiplied by the number of hours the data is stored in the month.
 
-Some of TiDB Cloud features are partially supported or not supported on TiDB Serverless. See [TiDB Serverless Limitations and Quotas](/tidb-cloud/serverless-limitations.md) for details.
+### Why does the storage used size remain unchanged after dropping a table or database immediately?
 
-### What can TiDB Serverless be used for?
+TiDB retains dropped tables and databases for a period of time to ensure that transactions that depend on these tables can continue to run smoothly. Moreover, the extended retention time enables TiDB to provide the [FLASHBACK TABLE](/sql-statements/sql-statement-flashback-table.md)/[FLASHBACK DATABASE](/sql-statements/sql-statement-flashback-database.md) feature, which allows you to recover dropped tables and databases in the event that they were mistakenly deleted.
 
-You can use your TiDB Serverless cluster for non-production workloads such as prototype applications, development environments, hackathons, and academic courses, or to provide temporary data service for your datasets.
+### Why are there RU consumptions when I'm not actively running any query?
 
-### I created a Developer Tier cluster before TiDB Serverless was available. Can I still use my cluster?
+RU consumptions can occur in various scenarios. One common scenario is during background queries, such as the synchronization of schema changes between TiDB instances. Another scenario is when certain web console features generate queries, like loading schemas. These processes utilize RUs even without explicit user triggers.
```

Comment:
```suggestion
RU consumptions can occur in various scenarios. One common scenario is during background queries, such as the replication of schema changes between TiDB instances. Another scenario is when certain web console features generate queries, like loading schemas. These processes utilize RUs even without explicit user triggers.
```

Reply:
Replication is more general but under this context I guess synchronization is what we used before. 
For example in the following document:
https://github.com/pingcap/tidb/blob/master/docs/design/2018-10-08-online-DDL.md#state-change-process

![image](https://github.com/pingcap/docs/assets/3982329/70a2e18c-79f0-4d79-ba8f-575af73e1e83)